colcon_core/subprocess.py

```diff
@@ -65,6 +66,8 @@ async def run(
     :param env: a dictionary with environment variables
     :param shell: whether to use the shell as the program to execute
     :param use_pty: whether to use a pseudo terminal
+    :param close_fds: whether descriptors except 0, 1 and 2 will be closed
+      before the child process is executed
```

Comment:
Either the documentation should also the `else` case (when the parameter is `False` or it should just reference the same parameter of the underlying Python API.